postgres-troubleshooting.md

```diff
@@ -0,0 +1,179 @@
+# Troubleshooting Postgres
+
+This guide assumes that postgres was installed using `brew`.
+
+## Is postgres running?
+
+To see if postgres is running, run:
+
+```
+brew services list
+```
+
+You should see something like:
+
+```
+Name          Status  User Plist
+postgresql    started edan /Users/edan/Library/LaunchAgents/homebrew.mxcl.postgresql.plist
+```
+
+If _Status_ does not show up as `started`, try starting or restarting postgres:
+
+```
+brew services restart postgresql
+# or
+brew services start postgresql
+```
+
+If `brew services list` continues to show an _error_ status, continue with the troubleshooting guide below.
+
+
+## Resolving common postgres errors
+
+### Have you tried turning it off and then back on again?
+
+Restart your computer! Then run:
+
+```
+# See if it's running again
+brew services list
+
+# If not, try restarting again
+brew services restart postgresql
+
+# Check to see if that fixed it
+brew services list
+```
+
+
+### postmaster.pid already exists
+
+Postgres sometimes has trouble if your computer shuts down unexpectedly (eg, out of battery). Deleting your `postmaster.pid` file can resolve this type of error. But first you need to find the file!
+
+Try:
+
+```
+ls /usr/local/var/postgres/postmaster.id
+
+# or
+
+ls /opt/homebrew/var/postgres/postmaster.pid
+
+# or
+
+ls $PGDATA/postmaster.pid
```

Comment:
This code block is new -- rather than looking in the error log, you can try some common locations, and see what works.